ckan/migration/__init__.py

```diff
@@ -0,0 +1,8 @@
+# encoding: utf-8
+
+
+def skip_based_on_legacy_engine_version(op, filename):
```

Comment:
Can you explain what this function does or what is needed for?

Reply:
just a docsting would be nice e.g. "graceful upgrade from old migration numbers to alembic ids"